### PR TITLE
Add cfitsio library to DALI deps

### PR DESCRIPTION
**Pull request by aderylo**
## Goal: 
- Provide tools for working with fits files.

### CODE REVIEW

File: build_scripts/build_cfitsio.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    echo "set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)" >> toolchain.cmake
+fi
+echo "set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")" >> toolchain.cmake
+echo "set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")" >> toolchain.cmake
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=toolchain.cmake \
+      -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
+      ..
+make -j"$(grep ^processor /proc/cpuinfo | wc -l)"
+make install
+popd
```
Comment (szalpal) [inline build_scripts/build_cfitsio.sh:39]:
```suggestion
popd

```

**Review by szalpal — APPROVED** [review]
Please remember to add the required acknowledgement

File: README.rst [hunk 2]
```diff
@@ -50,6 +50,8 @@ The repository consists mostly of externally hosted subrepositories:
 +----------------+---------------------+---------------------+
 | libtar_        | |libtarver|_        | |libtarlic|_        |
 +----------------+---------------------+---------------------+
+| cfitsio_        | |cfitsiover|_        | |cfitsiolic|_     |
```
Comment (mzient) [inline README.rst:53]:
```suggestion
| cfitsio_       | |cfitsiover|_      | |cfitsiolic|_       |
```

Comment (mzient) [inline README.rst:53]: Please align the table:
```suggestion
| libtar_        | |libtarver|_        | |libtarlic|_        |
+----------------+---------------------+---------------------+
| cfitsio_       | |cfitsiover|_       | |cfitsiolic|_       |
```

**Comment by klecki** [comment]
Build: 6680672

**Comment by klecki** [comment]
**CI MESSAGE**: [6705659]: BUILD STARTED

**Comment by klecki** [comment]
**CI MESSAGE**: [6707727]: BUILD STARTED - Xavier build, the deps should pass, the linter in Xavier will fail.

**Pull request merged**